planner_cspace/src/planner_3d.cpp

```diff
@@ -1299,28 +1302,51 @@ class Planner3dNode
       {
         updateStart();
 
-        if (jump_.detectJump())
+        static ros::Time last_time = start_.header.stamp;
+        const ros::Time time = start_.header.stamp;
+        if (time > last_time)
```

Comment:
I think this check can also be removed. (If needed, please replace `static` variable by member variable.)